### ISSUE TITLE
Make calendar indicator show all when notifications indicator is disabled.

### ISSUE THREAD

**Issue by SaucySauce** [Ordissimo/Big-Sur-StatusArea#6]
![Screenshot from 2021-03-21 18-42-54](https://user-images.githubusercontent.com/38703177/111918583-45b18280-8a75-11eb-9de8-b2175010389d.png)

As shown here, notifications indicator is disabled yet the calendar notification still acts as if notifications were being handled by another indicator. Also, the padding on the left border of the widget is too small compared to the right side.

**Comment by SaucySauce** [comment]
Also, the padding on the left border of the widget is too small compared to the right side.

**Comment by Ordissimo** [comment]
I have the same margins before and after activation of the extension.

![Capture d’écran du 2021-03-21 21-47-26](https://user-images.githubusercontent.com/10682892/111920438-3d664100-8a8f-11eb-90b2-899b5feb2bd7.png)
![Capture d’écran du 2021-03-21 21-46-30](https://user-images.githubusercontent.com/10682892/111920440-3d664100-8a8f-11eb-8419-4018c051a2a7.png)

**Comment by SaucySauce** [comment]
No, I mean, disabling just the notifications indicator.
![Screenshot from 2021-03-21 19-53-14](https://user-images.githubusercontent.com/38703177/111920545-1b64c280-8a7f-11eb-8bcb-b14bbe94f9ba.png)

**Comment by Ordissimo** [comment]
I don't understand the problem ....
If you have more precision?

**Comment by SaucySauce** [comment]
if big-sur-notifications-indicator: enable, big-sur-calendar-indicator: enable;

else stock-calendar/notifications: enable;

**Comment by Ordissimo** [comment]
Ok, I will correct this tonight!

**Comment by SaucySauce** [comment]
![Screenshot from 2021-03-26 21-11-08](https://user-images.githubusercontent.com/38703177/112697780-d32b1300-8e77-11eb-8b4c-a4e0a9f6353c.png)

What did you do? I see no change.

**Comment by SaucySauce** [comment]
This is what I mean
![Screenshot-20210326211507-954x120](https://user-images.githubusercontent.com/38703177/112698198-b17e5b80-8e78-11eb-804d-06f9cf718de8.png) when it is like this

The calendar should look like this (default gnome behavior)
![pic_5](https://user-images.githubusercontent.com/38703177/112698354-07eb9a00-8e79-11eb-8aa2-ba7b160e7b80.jpg)

**Comment by Ordissimo** [comment]
Ok, I didn't understand!
I have disabled system notifications as soon as you disable notifications.
As for what you want, it is to reintegrate the notifications in the calendar indicator, it can be done, but it is a feature and not a bug!
I'll look at how to do this in a way that preserves the native behavior, but also allows flexibility!

**Comment by SaucySauce** [comment]
Thanks. Yeah, I should've made my statements easier to understand. Sorry about that.

**Comment by Ordissimo** [comment]
Hi @SaucySauce ,Here is an implementation of the feature you asked for.
It's not perfect, the switch between the current version and your feature forces to restart gnome-shell, I haven't found another solution yet!
I took the opportunity to replace the icons by the system one.
Only one icon is still a problem, the bluetooth pairing.

**Comment by SaucySauce** [comment]
> Hi @SaucySauce ,Here is an implementation of the feature you asked for.
> It's not perfect, the switch between the current version and your feature forces to restart gnome-shell, I haven't found another solution yet!
> I took the opportunity to replace the icons by the system one.
> Only one icon is still a problem, the bluetooth pairing.

I'll check the code. Thanks.
Also, as for the Bluetooth pairing icon issue, you can request for the bluetooth paired icon and fallback to the standard active icon in case the system doesn't have a custom icon theme that has a `bluetooth-paired`.

**Comment by SaucySauce** [comment]
![Screenshot from 2021-04-03 13-33-21](https://user-images.githubusercontent.com/38703177/113479932-35d86c00-9481-11eb-852d-aa5612e38e22.png)

Still the same.

**Comment by Ordissimo** [comment]
Change branch.
```
git checkout asseble-date-and-notification
```
![parametre](https://user-images.githubusercontent.com/10682892/113480086-d8e5b180-9492-11eb-90ac-8d3df7ff5d9b.png)
Disable the Indicator XXXX option.
Reload gnome-shell

**Comment by SaucySauce** [comment]
oh ok. my mistake. Will check out soon.

**Comment by SaucySauce** [comment]
Ok, so it semi works, messing with the order which the indicators are enabled kills the function.

What if instead, you add a new indicator (just the default gnome shell calendar, clock, weather, notifications) with the ability to be moved in the order of indicators. I think it's preferable than to having to have the two, notifications and calendar, indicators communicate.

**Comment by Ordissimo** [comment]
This is not possible.
The problem is that we take the elements of gnome-shell and arrange them differently.
I don't play with the two indicators. I have a date indicator and the notification indicator.
When you disable the separation of the indicators the date indicator is disabled and it is the notification indicator that does both.
Tell me exactly what is wrong.

**Comment by SaucySauce** [comment]
The weather doesn't show.
Toggling either the notifications or calendar indicator after it's working will instantly kill the unified widget and go back to separate widgets.

**Comment by Ordissimo** [comment]
This does not work until gnome-shell is released.
A gnome-shell object cannot be used in two places.
I will check the weather. For the rest I can't do anything. I would have to restart the extension automatically.

**Comment by Ordissimo** [comment]
> Toggling either the notifications or calendar indicator after it's working will instantly kill the unified widget and go back to separate widgets.

Fixed

**Comment by SaucySauce** [comment]
Will verify. Thanks.

**Comment by SaucySauce** [comment]
It's better. Thanks.

**Comment by SaucySauce** [comment]
![Screenshot from 2021-04-03 16-45-36](https://user-images.githubusercontent.com/38703177/113485261-2c103200-949c-11eb-98b7-b0649bea215c.png)

Weird bug on the indicator itself.

**Comment by Ordissimo** [comment]
You talk about the date? Give details

**Comment by SaucySauce** [comment]
The weird gap on the right of the hour on the top bar.

**Comment by Ordissimo** [comment]
The space for the notification icon
```
notify-send test test1
```

**Comment by SaucySauce** [comment]
Can you make it dynamic so the gap isn't always there?

**Comment by Ordissimo** [comment]
> Can you make it dynamic so the gap isn't always there?

Fixed

**Comment by SaucySauce** [comment]
Excellent. Working perfectly.

**Comment by Ordissimo** [comment]
I watch tonight for the weather

**Comment by SaucySauce** [comment]
Thank you.

**Comment by SaucySauce** [comment]
https://github.com/Ordissimo/Big-Sur-StatusArea/pull/25

I did a small modification, don't know if it makes sense, Adwaita icons are pretty big but Newaita are smaller.

**Comment by Ordissimo** [comment]
Can you test it?

**Comment by SaucySauce** [comment]
![Screenshot from 2021-04-03 21-57-01](https://user-images.githubusercontent.com/38703177/113492549-98ecf180-94c7-11eb-8aae-256ca90aa136.png)
Has a blank gap underneath the weather but everything works fine.

**Comment by Ordissimo** [comment]
Can you test it?

**Comment by SaucySauce** [comment]
working perfectly.

**Comment by SaucySauce** [comment]
btw, can you make the date like this?
![Screenshot-20210404001557-379x373](https://user-images.githubusercontent.com/38703177/113494762-00f90300-94db-11eb-9d3a-a7d4efd48fbb.png)
it looks so pretty.

**Comment by Tuxman2** [comment]
Hi,

do you know where is defined this part (date above the calendar) in the gnome-shell.css file ? I'm searching it but I don't find it.

Thanks.

**Comment by Ordissimo** [comment]
> btw, can you make the date like this?

@SaucySauce, Fix with 99b26e7dcc8be72889f124d68e4fe28a63987d10

> do you know where is defined this part (date above the calendar) in the gnome-shell.css file ? I'm searching it but I don't find it.

@Tuxman2, look this  99b26e7dcc8be72889f124d68e4fe28a63987d10, the day-label class and the date-label class allow to define the style of the date

**Comment by SaucySauce** [comment]
Sweet. everything is working and looking good. I think this branch can be merged to master unless you think you can still improve on the notifications and calendar code.
